tests/api2/test_009_interface.py

```diff
@@ -148,25 +165,62 @@ def test_11_creating_vlan1_interface():
     interfaces_id = results.json()['id']
 
 
+@pytest.mark.skipif(ha, reason='Skiping test for HA')
 @pytest.mark.parametrize('dkey', ["ipv4_dhcp", "name", "vlan_parent_interface",
                                   "type", "vlan_tag", "vlan_pcp"])
 def test_12_compare_payload_with_created_vlan1_interface_result_output_(dkey):
     assert results.json()[dkey] == payload[dkey], results.text
 
 
+@pytest.mark.skipif(ha, reason='Skiping test for HA')
 def test_13_get_the_vlan1_interface_results():
     global results
     results = GET(f'/interface/id/{interfaces_id}/')
     assert results.status_code == 200, results.text
     assert isinstance(results.json(), dict) is True, results.text
 
 
+@pytest.mark.skipif(ha, reason='Skiping test for HA')
 @pytest.mark.parametrize('dkey', ["ipv4_dhcp", "name", "vlan_parent_interface",
                                   "type", "vlan_tag", "vlan_pcp"])
 def test_14_compare_payload_with_get_vlan1_interface_result_output_(dkey):
     assert results.json()[dkey] == payload[dkey], results.text
 
 
+@pytest.mark.skipif(not ha and "virtual_ip" not in os.environ, reason='Skiping test for Core')
```

Comment:
Before we configure networking, we need to ensure failover is disabled. (i'm not sure where that's being done)

Reply:
it is not done, probably good to add that.

Reply:
I'm going to DM you on the exact steps that should be taken when configuring an HA system as it's got a lot of steps that need to be taken in a specific order.

Reply:
@yocalebo I believe that disabling failover is not required for first time setup. https://jira.ixsystems.com/browse/NAS-105930

Reply:
Resolving this one after clarifying with @william-gr  "yes, disabling failover is not required on first setup".